src/events/api/events.ts

```diff
@@ -25,6 +25,11 @@ export const getEvents = async (args?: IEventAPIParameters): Promise<INewEvent[]
   return data.results;
 };
 
+export const getAllEventsParamtered = async (args: IEventAPIParameters): Promise<INewEvent[]> => {
+  const data = await getAllPages<INewEvent>(API_URL, { format: 'json', page_size: 80, ...args });
+  return data;
+};

```

Comment:
This could probably be combined with `getAllUserEvents` in a single `getAllEvents` function where the `user` is optional (`user?: IAuthUser`)